docs/Quickstart.md

```diff
@@ -9,7 +9,7 @@ Before you start using this SDK, read this requirements.
 - jq
 - Rust Installed
 - PHP Installed
-- FFI Extension is Enabled (_Why? I read the C heder definition from wrapper_)
+- FFI Extension is Enabled (_Why? I read the C header definition from wrapper_)
```

Comment:
Thank you and I appreciate this, but next time I will reject PRs with small sizes.